responsible-ai-collaborative/aiid
# Feature/email user submission

- closes #1759 

How to test
The easiest way to test this is to fork this branch and test it on your environment.
Don't forget to update your realm with the changes from this branch.

- Create a new submission while logged in
- Approve this submission

Once you approve a submission (Create a new incident based on a submission) you can trigger the mutation processNotifications to your realm
```
mutation ProcessNotifications {
      processNotifications
    }
```

Once you trigger this you should receive an email that your submission has been approved, with the link to the new incident.
